projects/packages/search/src/dashboard/components/mocked-search/mocked-instant-search.scss

```diff
@@ -1,4 +1,5 @@
-@import './_variables.scss';
+@import '../../scss/_variables.scss';
```

Comment:
By adding the root directory of the dashboard to our dashboard Webpack configuration, we will be able to use 
`@import 'scss/_variables.scss';` instead of `@import '../../scss/_variables.scss';`.

See the [Webpack config for Customberg for `resolve.modules`](https://github.com/Automattic/jetpack/pull/21637/files#diff-f2b5f73d87c464c22a35824a0ab093e0d7703f9d8c5bf5441cea7a607bcae505L30) as an example.

Reply:
hmm I do have it set here already: https://github.com/Automattic/jetpack/pull/21888/files#diff-93f7a0e8399381abf3246f3b56fbef9e3f17c8ae7a5758d19b403c68a14dc934R34 

addressed in 3fa8d7e8e5. Weird enough I tried to do this but failed, but it works now! thanks for the suggestion 👍